# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xine-lib/ffmpeg-api.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur xine-lib-1.1.15-orig/src/combined/ffmpeg/ff_dvaudio_decoder.c xine-lib-1.1.15/src/combined/ffmpeg/ff_dvaudio_decoder.c
--- xine-lib-1.1.15-orig/src/combined/ffmpeg/ff_dvaudio_decoder.c	2008-10-19 08:03:18.000000000 +0000
+++ xine-lib-1.1.15/src/combined/ffmpeg/ff_dvaudio_decoder.c	2008-10-19 08:31:03.000000000 +0000
@@ -53,7 +53,7 @@
 #endif
 
 #ifdef HAVE_FFMPEG_AVUTIL_H
-#  include <avcodec.h>
+#  include <ffmpeg/avcodec.h>
 #elif defined HAVE_FFMPEG
 #  include <libavcodec/avcodec.h>
 #else
diff -Nur xine-lib-1.1.15-orig/src/combined/ffmpeg/ffmpeg_decoder.h xine-lib-1.1.15/src/combined/ffmpeg/ffmpeg_decoder.h
--- xine-lib-1.1.15-orig/src/combined/ffmpeg/ffmpeg_decoder.h	2008-10-19 08:03:18.000000000 +0000
+++ xine-lib-1.1.15/src/combined/ffmpeg/ffmpeg_decoder.h	2008-10-19 08:20:33.000000000 +0000
@@ -26,7 +26,7 @@
 #endif
 
 #ifdef HAVE_FFMPEG_AVUTIL_H
-#  include <avcodec.h>
+#  include <libavcodec/avcodec.h>
 #elif defined HAVE_FFMPEG
 #  include <libavcodec/avcodec.h>
 #else
diff -Nur xine-lib-1.1.15-orig/src/combined/ffmpeg/ffmpeg_encoder.c xine-lib-1.1.15/src/combined/ffmpeg/ffmpeg_encoder.c
--- xine-lib-1.1.15-orig/src/combined/ffmpeg/ffmpeg_encoder.c	2008-10-19 08:03:18.000000000 +0000
+++ xine-lib-1.1.15/src/combined/ffmpeg/ffmpeg_encoder.c	2008-10-19 08:10:29.000000000 +0000
@@ -39,7 +39,7 @@
 #include "video_out_dxr3.h"
 
 #ifdef HAVE_FFMPEG_AVUTIL_H
-#  include <avcodec.h>
+#  include <ffmpeg/avcodec.h>
 #elif defined HAVE_FFMPEG
 #  include <libavcodec/avcodec.h>
 #else
